src/main/java/org/junitpioneer/jupiter/StdOutputStream.java

```diff
@@ -55,7 +55,7 @@ public String capturedString() {
 	 * @return the lines that were written to {@code System.out} or {@code System.err}
 	 */
 	public String[] capturedLines() {
-		var lines = writer.toString().split(StdIoExtension.SEPARATOR, -1);
+		var lines = out.toString(Charset.defaultCharset()).split(StdIoExtension.SEPARATOR, -1);
```

Comment:
Same thing here, `Charset.defaultCharset()` might be unnecessary?

Reply:
```suggestion
		var lines = out.toString().split(StdIoExtension.SEPARATOR, -1);
```